.template/addons/docker/docker-compose.test.yml.tt

```diff
@@ -19,14 +19,14 @@ services:
     build:
       context: .
       cache_from:
-        - ${DOCKER_IMAGE}:${BRANCH_TAG}
+        - ${DOCKER_REGISTRY_HOST}/${DOCKER_IMAGE}:${BRANCH_TAG}-test
       args:
         - BUILD_ENV=test
         - RUBY_ENV=test
         <%- if WEB_VARIANT -%>
         - NODE_ENV=test
         <%- end -%>
-    image: ${DOCKER_IMAGE}:${BRANCH_TAG}
+    image: ${DOCKER_REGISTRY_HOST}/${DOCKER_IMAGE}:${BRANCH_TAG}-test
```

Comment:
I'm curious why we need to create a separated `test` image? in `docker-compose.yml.tt` we're naming the image as 
```
image: ${DOCKER_REGISTRY_HOST}/${DOCKER_IMAGE}:${BRANCH_TAG}
```
isn't it enough? what is the benefit of this test image? 🤔

Reply:
The test image is often used for testing, the normal image is used for running the app.
Before, we usually use 1 name for both, but for me, we should create the separated images.
Both of them are pushed to the Docker registry, but if we use the same name, then 1 of them will be overridden by another.
So I think for each purpose, we should create different images.

Reply:
I think `docker-compose.test` is used for building test images (feature branches) while `docker-compose.yml` is used for production image (develop/master branch)
so even though they share the same naming pattern 
```yml
image: ${DOCKER_REGISTRY_HOST}/${DOCKER_IMAGE}:${BRANCH_TAG}
```
the images will never be overridden, IMO